src/Middleware/Diagnostics/src/DeveloperExceptionPage/Views/ErrorPage.cshtml

```diff
@@ -21,312 +21,341 @@
-                        {
-                            <tr>
-                                <td>@Resources.ErrorPageHtml_EndpointRouteOrder</td>
-                                <td>@Model.Endpoint.Order</td>
-                            </tr>
-                        }
-                        @if (!string.IsNullOrEmpty(Model.Endpoint.HttpMethods))
-                        {
-                            <tr>
-                                <td>@Resources.ErrorPageHtml_EndpointRouteHttpMethod</td>
-                                <td>@Model.Endpoint.HttpMethods</td>
-                            </tr>
-                        }
-                    </tbody>
-                </table>
-            }
-            else
-            {
-                <p>@Resources.ErrorPageHtml_NoEndpoint</p>
-            }
-            <h2>@Resources.ErrorPageHtml_RouteValues</h2>
-            @if (Model.RouteValues != null && Model.RouteValues.Any())
-            {
-                <table>
-                    <thead>
+                    }
+                </tbody>
+            </table>
+        }
+        else
+        {
+            <p>@Resources.ErrorPageHtml_NoEndpoint</p>
+        }
+        <h2>@Resources.ErrorPageHtml_RouteValues</h2>
+        @if (Model.RouteValues != null && Model.RouteValues.Any())
+        {
+            <table>
+                <thead>
+                    <tr>
+                        <th>@Resources.ErrorPageHtml_VariableColumn</th>
+                        <th>@Resources.ErrorPageHtml_ValueColumn</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    @foreach (var kv in Model.RouteValues.OrderBy(kv => kv.Key))
+                    {
                         <tr>
-                            <th>@Resources.ErrorPageHtml_VariableColumn</th>
-                            <th>@Resources.ErrorPageHtml_ValueColumn</th>
+                            <td>@kv.Key</td>
+                            <td>@(kv.Value!)</td>
                         </tr>
-                    </thead>
-                    <tbody>
-                        @foreach (var kv in Model.RouteValues.OrderBy(kv => kv.Key))
-                        {
-                            <tr>
-                                <td>@kv.Key</td>
-                                <td>@(kv.Value!)</td>
-                            </tr>
-                        }
-                    </tbody>
-                </table>
-            }
-            else
-            {
-                <p>@Resources.ErrorPageHtml_NoRouteValues</p>
-            }
-        </div>
-
-        <script>
-            //<!--
-            <%$ include: ErrorPage.js %>
-            //-->
-        </script>
-    </body>
+                    }
+                </tbody>
+            </table>
+        }
+        else
+        {
+            <p>@Resources.ErrorPageHtml_NoRouteValues</p>
+        }
+        <h2>@Resources.ErrorPageHtml_Metadata</h2>
```

Comment:
The endpoint metadata section is very related to the endpoint section. Either it should be:
- nested under the endpoint section, e.g. have an h3 or h4 title.
- or the metadata section should be next to endpoint and the title should be "Endpoint metadata`.